src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -217,7 +240,12 @@ DownloadOptionPanel.propTypes = {
 
 DownloadOptionPanel.defaultProps= {
     groupKey: 'DownloadDialog',
-};
+    updateRequest:null,
+    updateDownloadRequest:null,
+    dlParams:   PropTypes.shape({
+        SearchProcessor: undefined
+       })
```

Comment:
`updateRequest` should be `updateSearchRequest`
`dlParams` should not contains `shape`.  This is where you define default values.
Actually, they whole block of update is not needed.  It's setting default values that are either wrong or is already the default behavior.  I suggest removing the whole block.

Reply:
Thanks for spotting the left over stuff.  I changed the way to implement it and left those unused properties there.